content/articles/coming-soon-landing-page-using-javascript-timers/index.md

```diff
@@ -0,0 +1,463 @@
+### Introduction
+A `Coming Soon` landing page is a temporary home page that informs visitors that your website is in the process of being built or will be available soon. Since it lacks a header or a footer, visitors cannot navigate to the rest of your website hence it is important to capture what is under development so as to give as much information as possible.  
+
+Every coming soon page includes a statement encouraging visitors to return at a later time, as well as any other pertinent information concerning the website that is under development.  
+This article will teach you how to make a coming soon landing page but first things first you will learn about JavaScript timers to get you started.
+
+#### Table Of Contents
+- [Introduction](#introduction)
+- [JavaScript timers overview](#javascript-timers-overview)
+- [code execution after a delay](#code-execution-after-a-delay)
+- [Code Execution at regular intervals](#code-execution-at-regular-intervals)
+- [Cancelling a timer](#cancelling-a-timer)
+- [Template section](#template-section)
+- [Best practices](#best-practices)
+- [Conclusion](#conclusion)
+
+### Javascript timers overview
+A timer is a feature that allows us to run a program at a predetermined time. They can be used to postpone code execution so that it doesn't finish at the same time as an event or a page loading. For example, they can be used to alter your website's advertisement banners at regular intervals or to display a real-time clock, among other things.  
+
+In JavaScript, there are two timer functions: `setTimeout ()` and `setInterval ()`. Browsers implement timer functions, and their implementations differ from one browser to another. The Node.js runtime also has a native implementation of timers. In browsers, the window interface is the parent of the main timer features. Because the window interface makes its elements available globally in the main javascript scope, setTimeout may be used in the browser console.
+'Timers' is an object in NodeJs that functions like the browser's window interface.

```

Comment:
Note the changes I've made in this section. Make your article flow. Break sentences down to make it easy to follow along. Do not throw in words like 'Node.js runtime` . What does it do? Make the article as simple as possible where applicable, also, breakdown paragraphs. Refrain from wordy paragraphs!